raddb/mods-available/ldap

```diff
@@ -135,231 +191,287 @@ ldap {
 		#
-		#  If the value of the retrieved access_attribute is
-		#  'false', it will negate the result.
+		#  If the value of the retrieved `access_attribute` is `false`, it will
+		#  negate the result.
 		#
-		#  e.g.
+		#  e.g:
+		#
+		#  ```
 		#    access_positive = yes
 		#    access_attribute = userAccessAllowed
+		#  ```
 		#
 		#  With an LDAP object containing:
+		#
+		#  ```
 		#    userAccessAllowed: false
+		#  ```
 		#
 		#  Will result in the user being locked out.
+		#  ====
+		#
 #		access_positive = yes
 	}
 
 	#
-	#  User membership checking.
+	#  ### User membership checking
 	#
 	group {
-		#  Where to start searching in the tree for groups
+		#
+		#  base_dn:: Where to start searching in the tree for groups.
+		#
 		base_dn = "${..base_dn}"
 
-		#  Filter for group objects, should match all available
+		#
+		#  filter:: Filter for group objects, should match all available
 		#  group objects a user might be a member of.
 		#
-		#  If using Active Directory you are likely to need "group"
-		#  instead of "posixGroup".
+		#  If using Active Directory you are likely to need `group`
+		#  instead of `posixGroup`.
+		#
 		filter = '(objectClass=posixGroup)'
 
-		# Search scope, may be 'base', 'one', sub' or 'children'
+		#
+		#  scope:: Search scope, may be `base`, `one`, `sub` or `children`.
+		#
 #		scope = 'sub'
 
-		#  Attribute that uniquely identifies a group.
-		#  Is used when converting group DNs to group
-		#  names.
+		#
+		#  name_attribute:: Attribute that uniquely identifies a group.
+		#
+		#  Is used when converting group DNs to group names.
+		#
 #		name_attribute = cn
 
-		#  Filter to find all group objects a user is a member of.
-		#  That is, group objects with attributes that
-		#  identify members (the inverse of membership_attribute).
+		#
+		#  membership_filter:: Filter to find all group objects a user is a member of.
+		#
+		#  That is, group objects with attributes that identify
+		#  members (the inverse of `membership_attribute`).
+		#
 #		membership_filter = "(|(member=%{control:Ldap-UserDn})(memberUid=%{%{Stripped-User-Name}:-%{User-Name}}))"
 
-		#  The attribute, in user objects, which contain the names
-		#  or DNs of groups a user is a member of.
+		#
+		#  membership_attribute:: The attribute, in user objects, which contain
+		#  the names or DNs of groups a user is a member of.
 		#
 		#  Unless a conversion between group name and group DN is
 		#  needed, there's no requirement for the group objects
 		#  referenced to actually exist.
 		#
-		#  If the LDAP server does not support the "memberOf"
-		#  attribute (or equivalent), then you will need to use the
-		#  membership_filter option above instead. If you can't see
-		#  the memberOf attribute then it is also possible that the
-		#  LDAP bind user does not have the correct permissions to
-		#  view it.
+		#  If the LDAP server does not support the `memberOf` attribute (or equivalent),
```

Comment:
I'd say that is noteworthy.  i.e. hints about interoperating with things outside of FreeRADIUS